### PR TITLE
GUI improvements

### LINKED ISSUE #762: GUI changes
- [x] Change 'learn more' to two buttons 'About us' and 'How to use BIP' and remove the buttons at the top.
- [x] Trial submission: Change font style in step 3 “Origin of genetic material used in this trial “ to the same style as “Design factors”
- [x] Similar to the trial submission, the population submission will have a clearly visible “Download Template” button.

### PR DESCRIPTION
**Pull request by kammerer**
Fixes #762

### CODE REVIEW

File: app/views/submissions/steps/trial/_step04_before.html.haml [hunk 1]
```diff
@@ -91,7 +93,6 @@
         Please use the values from the <b>Variety</b> column when referring to existing plant varieties
         in your CSV file in order to avoid duplications due to differences in spelling.
 
-  %br
   %h5 Trait scores ordering #{'and technical replicates' if rep_traits.present?}
   %p
     The first line will be treated as a header. Specific values in columns
```
Comment (Nuanda) [inline app/views/submissions/steps/trial/_step04_before.html.haml]: There is something wrong with column ordering in this info box. In my case it says the trait scoring starts in column 5 while the template has its first trait scoring column at index 6 (starting with index 1, as a user would do). Please check again if we are counting these columns correctly in this info box.

Comment (kammerer) [reply app/views/submissions/steps/trial/_step04_before.html.haml]: You are right, the numbers were not updated when we added "Year produced" column.

File: app/helpers/submissions_helper.rb [hunk 2]
```diff
@@ -85,6 +85,13 @@ def delete_submission_upload_button(upload, options = {})
     link_to "Delete", url, options
   end
 
+  def submission_template_link(path)
+    link_to path, class: 'btn btn-success btn-lg' do
```
[the remark this answers is not in the document]
Comment (kammerer) [reply app/helpers/submissions_helper.rb]: There was no such request but I think that non-standard size better draws the user's attention (and does not look bad):

* big - https://screenshots.firefox.com/JieffTbNgKUBuBra/lvh.me
* normal - https://screenshots.firefox.com/saw1JGrK4oiHmx0D/lvh.me
 
@wjurkowski, which one do you prefer?

[the remark this answers is not in the document]
Comment (kammerer) [reply app/helpers/submissions_helper.rb]: I want to merge this PR, and I will go with normal-sized button for now.

[the remark this answers is not in the document]
Comment (Nuanda) [reply app/helpers/submissions_helper.rb]: Sure. I've already approved changes last week, so you are free to merge as you please.

**Pull request merged**